buildcheck/backend/models.py

```diff
@@ -0,0 +1,73 @@
+class Point:
+    def __init__(self, x: float, y: float):
+        self.x = x
+        self.y = y
+
+class Edge:
+    def __init__(self, start: Point, end: Point):
+        self.start = start
+        self.end = end
```

Comment:
The names `start` and `end` carry some kind of connotation about ordering, which we don't guarantee as of yet.

Let's refactor this to `Edge(a: Point, b:Point)`.